Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/ui/main/MainActivity.java

```diff
@@ -163,8 +163,15 @@ public void launchSettingsActivity(MenuTag menuTag)
   @Override
   public void launchFileListActivity()
   {
-    Intent intent = new Intent(Intent.ACTION_OPEN_DOCUMENT_TREE);
-    startActivityForResult(intent, MainPresenter.REQUEST_DIRECTORY);
+    if (DirectoryInitialization.preferOldFolderPicker(this))
```

Comment:
Any opinion on moving this into openDirectoryPicker?

Reply:
That would be inappropriate, because the `openDirectoryPicker` call from `SettingsAdapter` must not use the new file picker. I suppose I could create a second variant of `openDirectoryPicker` for this, but it somewhat obscures the logic of how the intent the activity returns as a result should be handled (whether `getSelectedPath` should be used or not).